lienflash/lienflash-fe
# Tm/login feature

#### What’s this PR do?
- Create a landing page for not logged in users
- Create a login screen for users
- Change routing for logged in vs. logged out
- Add log out button to header
- Fixed api calls
- Fixed reducers w/changes to BE
- Add and refactor all tests

#### Where should the reviewer start?
- Everywhere/anywhere

#### How should this be manually tested?
- in terminal, run npm test to see all passing tests
- in terminal, run npm start; to login, use email1234@gmail.com, password1

#### Any background context you want to provide?
- This is a stretch goal so pretty happy we got to this point
- Had to adjust a lot of tests due to changes with adding this feature

#### What are the relevant tickets?
resolves #55 resolves #56 resolves #62 resolves #63 

#### Need Help/Support
- Going to need help with getting full test coverage
